public/app/core/navigation/GrafanaRouteLoading.tsx

```diff
@@ -1,24 +1,40 @@
-import { css } from '@emotion/css';
+import { css, cx } from '@emotion/css';
 import React from 'react';
 
-import { LoadingPlaceholder, useStyles2 } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { useStyles2 } from '@grafana/ui';
 
 export function GrafanaRouteLoading() {
   const styles = useStyles2(getStyles);
 
   return (
-    <div className={styles.loadingPage}>
-      <LoadingPlaceholder text={'Loading...'} />
+    <div
+      className={cx('preloader', styles.loadingPage, {
+        [styles.loadingPageDockedNav]: config.featureToggles.dockedMegaMenu,
+      })}
+      aria-live="polite"
+      role="status"
+      aria-label="Loading"
```

Comment:
can you translate this pwease? 🥺

Reply:
oh yeah whoops, that was on my mental todo list and i just forgot 😂